### ISSUE TITLE
Request for access for ML-Based DoS Attack Detection Training on CICDDoS2019 Dataset starting with Random Forest Tuning Hyperparameters

### ISSUE THREAD

**Issue by arifhakimi** [malaysia-ai/Jupyterhub#18]
What is your name?
Muhammad Arif Hakimi

What is your title?
PhD Researcher in Cybersecurity and Network Communications in IoV (Universiti Teknologi Malaysia)

What can you help with?
My research is centered on applying machine learning techniques to enhance DoS attack detection in IoV-SDN environments. I am currently focusing on training and fine-tuning a Random Forest classifier on the CICDDoS2019 dataset, with an emphasis on hyperparameter optimization to improve detection accuracy and speed, which are critical for real-time vehicular communication systems.

Additional context about yourself (please brag)
I am a seasoned researcher with a PhD in cybersecurity, specializing in vehicular communication networks. My expertise includes applying advanced data analytics and machine learning algorithms to network security challenges. I have published paper (some are still publishing and to be publish for next cybersecurity conference) on security in vehicular networks. My research has contributed to the development of innovative security solutions for next-generation vehicular networks, garnering recognition from both academic peers and industry leaders. With access to powerful computing resources, I am poised to significantly advance the state of the art in DoS attack mitigation with greater accuracy, ensuring safer and more secure vehicular communications in the era of Internet of Vehicles (IoV).

**Comment by huseinzol05** [comment]
invited to the org, this is required because we use github oauth2 app to secure proxy

**Comment by huseinzol05** [comment]
<img width="981" alt="Screenshot 2023-11-28 at 1 45 38 PM" src="https://github.com/malaysia-ai/jupyter-gpu/assets/19810909/d86f8fc1-2bac-40b1-991c-ac04fe68faad">

96 cores, https://jupyter4.us.mesolitica.com, good luck.

**Comment by arifhakimi** [comment]
Thanks. Is this the only way to access the terminal?

<img width="228" alt="image" src="https://github.com/malaysia-ai/jupyter-gpu/assets/53549510/5c4e59c5-ef00-4cf4-a811-4149afb0c7f3">

Do we have SSH access since we are using github oauth2 app to secure proxy?

**Comment by huseinzol05** [comment]
no, i did not bind external ip and open ssh port, u can use VS Code provided

**Comment by arifhakimi** [comment]
Assalamualaikum Mr Husein, Ms. Aisyah,

Looks like I got this error message lately "404 Not Found" when trying to access https://jupyter4.us.mesolitica.com

<img width="248" alt="image" src="https://github.com/malaysia-ai/jupyter-gpu/assets/53549510/6d0b3db5-93cf-4c08-9c00-207532777d8e">

I am trying to access this spot in order to execute the training on DoS attack signature using RF Algorithm.

 @huseinzol05, @aisyahrzk
